sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -716,24 +951,38 @@ private static async IAsyncEnumerable<PartitionEvent> MockPartitionEventEnumerab
         ///   Serves as a non-functional connection for testing consumer functionality.
         /// </summary>
         ///
-        private class MockConnection : EventHubConnection
+        internal class MockConnection : EventHubConnection
```

Comment:
If we're using these outside the context of this class, we should probably look to move them to a dedicated class or set of classes under `Infrastructure`.